static/styles/lib/loggedin.scss

```diff
@@ -203,7 +203,7 @@ aside.nav-sidebar {
 				height: 60px;
 				color: $secondaryColor;
 				fill:$secondaryColor;
-				padding: 0px 20px;
+				padding: 0px 5px;
```

Comment:
does this not produce any sideffects in a desktop view?

Reply:
No,  this amout of pixeles was killing the svg icon task.... all of those items in the list are icons but the task is an svg maybe they couldnt find the proper icon and they put this svg.  With 20px this svg disapear from the screen because is too much in small devices.  But  doesn't represent side effect to the siblings icon elements.